### DISCUSSION TITLE
Taro 3.x Video 文档书写有误，onFullscreenChange，onWaiting的说明写成一样的了

### DISCUSSION THREAD

**Discussion by huanghly** [NervJS/taro#12597] [Q & A]
<img width="1360" alt="image" src="https://user-images.githubusercontent.com/28948944/192927737-deee0750-8ffe-41f9-860f-53f3f2065b31.png">

**Comment by Chen-jj** [comment]
@huanghly 点文档下方的编辑按钮直接 PR 帮忙修复下咯~
